packages/@aws-cdk/aws-servicecatalogappregistry/README.md

```diff
@@ -191,6 +230,16 @@ declare const attributeGroup: appreg.AttributeGroup;
 application.associateAttributeGroup(attributeGroup);
 ```
 
+### Associating an attribute group with application
+
+You can associate an application with an attribute group with `associateApplicationWithAttributeGroup`:
+
+```ts
+declare const application: appreg.Application;
+declare const attributeGroup: appreg.AttributeGroup;
+attributeGroup.associateApplicationWithAttributeGroup(application);
```

Comment:
This is good, although the method name is a little long. Given that the subject is of type `AttributeGroup` and the object is of type `Application`, do you really think we need the words "Application" and "AttributeGroup" in the method name? 

Why not

```ts
attributeGroup.associate(application);
// or
attributeGroup.associateWith(application);
// or
attributeGroup.associateTo(application);
```

Reply:
Yeah `associateWith` makes a lot more sense. Changed the same in the latest version.